docs/source/index.rst

```diff
@@ -1,10 +1,14 @@
-.. Podman documentation master file, created by
-   sphinx-quickstart on Tue Oct 22 15:20:30 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. include:: includes.rst
 
-Welcome to Podman's documentation!
+What is Podman?
 ==================================
+Podman_ is a daemonless, open source, Linux native tool designed to make it easy to find, run, build, share and deploy applications using Open Containers Initiative (OCI_) Containers_ and `Container Images`_. Podman provides a command line interface (CLI) familiar to anyone who has used the Docker `Container Engine`_. Most users can simply alias Docker to Podman (`alias docker=podman`) without any problems. Similiar other common `Container Engines`_ (Docker, CRI-O, containerd), Podman relies on an OCI compliant `Container Runtime`_ (runc, crun, runv, etc) to interface with the operating system and create the running containers.This makes the running containers created by Podman nearly indistinguishable from those created by any other common container engine.
```

Comment:
"Similar other common" -> "Similar to other common"

Reply:
Fixed

Reply:
I'm not seeing the fix, still looks like the old text.  Did git bite you?